plugins/jenkins/src/components/BuildsPage/lib/CITable/CITable.tsx

```diff
@@ -173,13 +175,48 @@ const generatedColumns: TableColumn[] = [
   {
     title: 'Actions',
     sorting: false,
-    render: (row: Partial<Project>) => (
-      <Tooltip title="Rerun build">
-        <IconButton onClick={row.onRestartClick}>
-          <RetryIcon />
-        </IconButton>
-      </Tooltip>
-    ),
+    render: (row: Partial<Project>) => {
+      const ActionWrapper = () => {
+        const [isLoadingRebuild, setIsLoadingRebuild] = useState(false);
+        const alertApi = useApi(alertApiRef);
+
+        const onRebuild = async () => {
+          if (row.onRestartClick) {
+            setIsLoadingRebuild(true);
+            try {
+              await row.onRestartClick();
+              alertApi.post({
+                message: 'Jenkins re-build has successfully executed',
+                severity: 'success',
+              });
+            } catch (e) {
+              if (e instanceof ResponseError) {
```

Comment:
```suggestion
              if (e.name === 'ResponseError') {
```

is the better check! And if you do this, you can actually remove the import at the top :)

But come to think of it ... do you really need to check the type of error at all? All errors have a `.message` anyway.

Reply:
At this moment for sure it does not make a check here, I thought it would be good to maintenance the code and if someone wants to add additional information there, it can be done easily :) But if it does not with conventions, I can remove it.

Reply:
@freben just pushed new changes, added @backstage/erros to dep. list, I hope it is fine for you :)